ods_ci/tests/Tests/400__ods_dashboard/412__ods_dashboard_resources.robot

```diff
@@ -268,24 +270,19 @@ Set Expected Items Based On RHODS Type    # robocop: disable
     ...    Jupyter
     ${EXPECTED_ITEM_PROVIDERS}=    Create List       by Anaconda Professional
     ${EXPECTED_ITEM_RESOURCE_TYPE}=    Create List     Tutorial
-    ${EXPECTED_ITEMS_FOR_PROVIDER_TYPE}=    Create List
-    ...    Creating a Jupyter notebook
-    ...    How to install Python packages on your notebook server
-    ...    How to update notebook server settings
-    ...    How to use data from Amazon S3 buckets
-    ...    How to view installed packages on your notebook server
-    ...    Deploying a sample Python application using Flask and OpenShift.
-    ...    Jupyter
+    ${EXPECTED_ITEMS_FOR_PROVIDER_TYPE}=     Evaluate    ${EXPECTED_ITEMS_FOR_ENABLE}.copy()
+    Append To List    ${EXPECTED_ITEMS_FOR_PROVIDER_TYPE}
     ...    OpenShift AI tutorial - Fraud detection example
-    ...    OpenShift API Management
-    ...    Securing a deployed model using Red Hat OpenShift API Management
+    ...    Red Hat OpenShift AI
     @{EXPECTED_ITEMS_FOR_COMBINATIONS}=      Create List
-    ...    Jupyter    OpenShift API Management
+    ...    Jupyter
+    ...    Red Hat OpenShift AI
     IF    ${is_self_managed} == ${TRUE}
-        Remove From List   ${EXPECTED_ITEMS_FOR_PROVIDER_TYPE}   -1
-        Remove From List   ${EXPECTED_ITEMS_FOR_PROVIDER_TYPE}   -1
-        Remove From List   ${EXPECTED_ITEMS_FOR_COMBINATIONS}   -1
-        ${n_items}=    Set Variable    46
+        # Deprecated
+        # Remove From List   ${EXPECTED_ITEMS_FOR_PROVIDER_TYPE}   -1
+        # Remove From List   ${EXPECTED_ITEMS_FOR_PROVIDER_TYPE}   -1
+        # Remove From List   ${EXPECTED_ITEMS_FOR_COMBINATIONS}   -1
```

Comment:
What does it mean - deprecated and commented out? Shall we remove it?

Reply:
This was made for SM clusters, but it is not relevant anymore. We might find it later required to re-add, so I just commented it out as deprecated.